docs/porting/third-party-deps.md

```diff
@@ -30,24 +30,29 @@ netstandard1.4
 netstandard1.5
 netstandard1.6
 netcoreapp1.0
-portable-<tfm_list>-whatever
+portable-net45-win8
+portable-win8-wpa8
+portable-net451-win81
+portable-net45-win8-wpa8-wpa81
 ```
 
-These are the Target Framework Monikers (TFM for short) which map to versions of [The .NET Standard Library](../standard/library.md).  Note that `netcoreapp1.0`, while compatible, is for applications and not libraries.  Although there's nothing wrong with using a library which is `netcoreapp1.0`-based, that library may not be intended for anything *other* than consumption by other `netcoreapp1.0` applications.
+These are the Target Framework Monikers (TFM for short) which map to versions of [The .NET Standard Library](../standard/library.md) and traditional Portable Class Library (PCL) profiles which are compatible with .NET Core.  Note that `netcoreapp1.0`, while compatible, is for applications and not libraries.  Although there's nothing wrong with using a library which is `netcoreapp1.0`-based, that library may not be intended for anything *other* than consumption by other `netcoreapp1.0` applications.
 
 There are also some legacy TFMs used in pre-release versions of .NET Core that may also be compatible:
 
 ```
 dnxcore50
-dotnet1.0
-dotnet1.1
-dotnet1.2
-dotnet1.3
-dotnet1.4
-dotnet1.5
+dotnet5.0
+dotnet5.1
+dotnet5.2
+dotnet5.3
+dotnet5.4
+dotnet5.5
 ```
```

Comment:
One thing I didn't notice before: according to [the *.NET Platform Standard* page](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#deprecated-monikers), it's `dotnet5.1` - `dotnet5.6`, not 5.0 - 5.5.